textBasedAdventure.js

```diff
@@ -3,3 +3,131 @@ const readline = require('readline-sync')
 let nameInput = readline.question("Enter your name: ")
 
 console.log(`Hello ${nameInput}!  Welcome to my game.`)
+
+//1.Create a video game
+let readlineSync = require('readline-sync');
+let i =""
+
+
+while(i !== 'y' || i === 'n'){
+    i = readlineSync.question("Your task is to create a video game. Would you like to create one?(y/n)")
+   
+ if(i === 'y') {
+     //'Y' key was pressed
+  console.log("You accepted the task!");
+  break
+ } 
+  if (i === 'n') {
+     //'N' was pressed
+  console.log("You rejected the task. (Answer yes)")
+ }
+}
+
+//2.How will you spend your time
+let readlineSync1 = require('readline-sync'),
+  decisions = ["Start the day before it's due", 'Start now', 'Ask someone for help'],
+  index = readlineSync1.keyInSelect(decisions, 'You only have 2 weeks to complete make a video game. What will you do next?');
+console.log(" You decided to " + decisions[index]);
+
+//3.You need help
+let readlineSync2 = require('readline-sync')
+
+for(a = 0; a <= Infinity; a++)
```

Comment:
By convention we always use `i` and `i` needs to be declared: `for (let i = 0...)`